examples/oft-solana/tasks/evm/send.ts

```diff
@@ -38,17 +38,16 @@ const action: ActionType<TaskArguments> = async ({ dstEid, amount, to }, hre: Ha
         composeMsg: '0x',
         oftCmd: '0x',
     }
-
-    console.dir({ sendParam }, { depth: null })
-
     const [msgFee] = await token.functions.quoteSend(sendParam, false)
-    console.dir(msgFee)
     const txResponse = await token.functions.send(sendParam, msgFee, signer.address, {
         value: msgFee.nativeFee,
         gasLimit: 500_000,
     })
     const txReceipt = await txResponse.wait()
     console.log(`send: ${amount} to ${to}: ${txReceipt.transactionHash}`)
+    console.log(
+        `Track cross-chain transfer here: ${getLayerZeroScanLink(txReceipt.transactionHash, dstEid == EndpointId.SOLANA_V2_TESTNET)}`
+    )
```

Comment:
For later - we have the nicer loggers, we should replace these console.logs with those

Reply:
Good point. Will do in follow-up.